relayer/relays/beacon/header/syncer/syncer.go

```diff
@@ -68,19 +68,11 @@ type SyncAggregate struct {
 func (s *Syncer) GetSyncPeriodsToFetch(lastSyncedPeriod, currentSlot uint64) ([]uint64, error) {
 	currentSyncPeriod := s.ComputeSyncPeriodAtSlot(currentSlot)
 
-	//The current sync period's next sync committee should be synced too. So even
-	// if the syncing is up-to-date with the current period, we still need to sync the current
-	// period's next sync committee.
-	if lastSyncedPeriod == currentSyncPeriod {
-		return []uint64{currentSyncPeriod}, nil
+	if lastSyncedPeriod < currentSyncPeriod {
+		return []uint64{lastSyncedPeriod + 1}, nil
 	}
 
-	syncPeriodsToFetch := []uint64{}
-	for i := lastSyncedPeriod; i <= currentSyncPeriod; i++ {
-		syncPeriodsToFetch = append(syncPeriodsToFetch, i)
-	}
-
-	return syncPeriodsToFetch, nil
+	return []uint64{}, nil
```

Comment:
Change algorithms here as we've already store `NextSyncCommittee` in beacon client so no need to sync `currentSyncPeriod` any more.